search/django/weedid/utils.py

```diff
@@ -118,3 +147,20 @@ def retrieve_listing_info(query_entity, awaiting_review):
         "contributor": query_entity.user.username,
         "contributor_email": query_entity.user.email if awaiting_review else "",
     }
+
+
+def retrieve_category_name(category):
+    if re.fullmatch(r"(crop|weed): ([a-z][a-z ]+|UNSPECIFIED)", category["name"]):
```

Comment:
A scientific name can have non-alphabetic characters in it, notably characters like `ö`. I think it's fine to test if it begins `crop` or `weed`.

We also allow the category name to be "other".

Reply:
Should I add "other" to the category schema? It's currently missing.